apps/src/aiTutor/redux/aiTutorRedux.ts

```diff
@@ -40,8 +39,18 @@ const initialState: AITutorState = {
 
 export interface ChatContext {
```

Comment:
Would it make sense to move `ChatContext` and `GeneralChatContext` into `aiTutor/types`?